### ISSUE TITLE
Ingest clinical trials 

### ISSUE THREAD

**Issue by lmeyerov** [TheDataRideAlongs/ProjectDomino#26]
-- Load WHO + clinicaltrails.gov
-- write into  Neo4j as model: registration date, trial start date, trial end date, drug, tags (COVID?), database, ...

**Comment by 007vasy** [comment]
these dates could be added to the current wf next to the drugs and synonyms and studies WF

**Comment by lmeyerov** [comment]
Yeah maybe the thing to do is start on analysis here first -- look at some concrete cases of ~popular URL references -- and figure out how to make a trustworthiness decision.
